sycl/source/detail/pi.cpp

```diff
@@ -139,14 +141,26 @@ uint64_t emitFunctionWithArgsBeginTrace(uint32_t FuncID, const char *FuncName,
 #ifdef XPTI_ENABLE_INSTRUMENTATION
   if (xptiTraceEnabled()) {
     uint8_t StreamID = xptiRegisterStream(SYCL_PIDEBUGCALL_STREAM_NAME);
-    CorrelationID = xptiGetUniqueId();
 
     xpti::function_with_args_t Payload{FuncID, FuncName, ArgsData, nullptr,
                                        &Plugin};
+    {
+      detail::tls_code_loc_t Tls;
+      auto CodeLoc = Tls.query();
+      xpti::payload_t PL = xpti::payload_t(
+          CodeLoc.functionName(), CodeLoc.fileName(), CodeLoc.lineNumber(),
+          CodeLoc.columnNumber(), nullptr);
+      uint64_t InstanceNumber{};
+      assert(GPIArgCallActiveEvent == nullptr);
+      GPIArgCallActiveEvent =
+          xptiMakeEvent("Plugin interface call", &PL, xpti::trace_graph_event,
+                        xpti_at::active, &InstanceNumber);
+    }
 
+    CorrelationID = xptiGetUniqueId();
     xptiNotifySubscribers(
         StreamID, (uint16_t)xpti::trace_point_type_t::function_with_args_begin,
-        GPIArgCallEvent, nullptr, CorrelationID, &Payload);
+        GPIArgCallEvent, GPIArgCallActiveEvent, CorrelationID, &Payload);
```

Comment:
to check if we need to update xpti documentation

Reply:
https://github.com/intel/llvm/pull/9840/commits/dbee52f5456e4aed56d60986abd2b49bb1f1ad9f